cmake/DaemonFlags.cmake

```diff
@@ -148,13 +148,20 @@ else()
     set(WARNMODE "no-")
 endif()
 
+# Compiler options
+option(USE_FLOAT_EXCEPTIONS "Use floating point exceptions" OFF)
```

Comment:
The use description of this is not quite accurate. It doesn't turn on exceptions; it alters compiler options in a way that makes the exceptions more likely to be useful. Maybe you could call it USE_FLOAT_DEBUG_MODE or something

Reply:
Actually I think the name is fine but it would be good to mention the cvars. `Enable floating point exceptions with common.floatException.* cvars`